react/src/components/Weather/CurrentWeather.js

```diff
@@ -5,28 +5,32 @@ import rain from '../../icons/desktop/weatherrain.png';
 import cloud from '../../icons/desktop/weathercloud.png';
 import { format } from 'date-fns';
 import { pl } from 'date-fns/locale'
-
+import { DisplayPressure,
+DisplayClouds,
+DisplayFeelTemp,
+DisplayWind,
+DisplayTemp,
+DisplaySmog, 
+DisplayHumidity,
+DisplaySunrise,
+DisplaySunset, } from './APIValidation'
+import { render } from 'react-dom';
 
 const icons = {
     'Clear': sun,
     'Rain': rain,
     'Clouds': cloud
 };
-
 const DisplayCity = ({city}) => {
     return (!!city) ?
          ( <div className={CurrentStyle.location}>{city}</div> ) :
-         (<></>)
+         (<>N/A</>)
 }
```

Comment:
To identyczny komponent do omawianego wyżej `Data`. Jeśli chcesz tylko do jednego spana przesłać style, możesz po prostu przesłać propa `customClassName` do funkcji `Data`, a komponent `Data` zaimportować do tego pliku, zamiast tworzyć tutaj duplikat.

`const Data = ({ data, customClassName }) => <span className={customClassName}>{!!data ? data : 'N/A'}</span>`